.github/workflows/main.yml

```diff
@@ -43,9 +43,13 @@ jobs:
           fetch-depth: 0
 
       # Install dependencies
+      - name: List all current packages
+        run: sudo apt --installed list
       - name: Remove pre-existing dotnet sdk 8 installs
-        run: sudo apt remove --purge dotnet-sdk-8.0
+        run: sudo apt remove --purge '^dotnet-.*' '^aspnetcore-.*'
```

Comment:
This change wasn't ultimately necessary but is more complete. Should share on the MS github issue around this stuff